service/docs/source/install.rst

```diff
@@ -9,10 +9,14 @@ Build Service.
 
 The packages built from this service with each update are:
```

Comment:
To help avoid confusion w/ the overloaded term 'service', replace "built from this service with... " --> "built from OBS with..."
Also, add "(OBS)" at the end of the last sentence in the previous paragraph.